ann_cnn_training/training.py

```diff
@@ -114,6 +124,9 @@ def write_log(*args):
 write_log(
     f"Training the {domain} horizontal and {vertical} vertical model, with features {features} with min-max learning rates {lr_min} to {lr_max}, and dropout={dropout}. Starting from epoch {init_epoch}. Training on years {train_years} and testing on years {test_years}.\n"
 )
+
+# write_log(train_files)
+# write_log(test_files)
 write_log("Defined input files")
 write_log(f"train batch size = {bs_train}")
 write_log(f"validation batch size = {bs_test}")
```

Comment:
similar to my prev comment. I would use `logging` rather than custom log functions